src/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="EnsureNETCoreAppRuntime" 
+          Condition="'$(RuntimeIdentifier)' != '' AND '$(EnsureNETCoreAppRuntime)' != 'false'" 
+          AfterTargets="RunResolvePackageDependencies">
+    <ItemGroup>
+      <_netCoreAppPackageDependencies 
+          Include="@(PackageDependencies->WithMetadataValue('ParentPackage', '$(MSBuildThisFileName)/$(_Microsoft_NETCore_App_Version)'))" />
+      <_activeRIDNetCoreAppPackageDependencies 
+          Include="@(_netCoreAppPackageDependencies->WithMetadataValue('ParentTarget', '$(NuGetTargetMoniker)/$(RuntimeIdentifier)'))" />
+      <_activeTFMNetCoreAppPackageDependencies 
+          Include="@(_netCoreAppPackageDependencies->WithMetadataValue('ParentTarget', '$(NuGetTargetMoniker)'))" />
+      <_ridSpecificNetCoreAppPackageDependencies 
+          Include="@(_activeRIDNetCoreAppPackageDependencies)" 
+          Exclude="@(_activeTFMNetCoreAppPackageDependencies)" />
+    </ItemGroup>
+    
+    <Error Condition="'@(_ridSpecificNetCoreAppPackageDependencies)' == ''"
```

Comment:
This target file only gets included for MS.NETCore.App.  For cases like Tizen I'd expect them to parent that package in their runtime.json, even if they provide runtime.json in a separate package.  Otherwise they can't replace our runtime mappings.